integration/embed_test.go

```diff
@@ -115,3 +117,34 @@ func setupEmbedCfg(cfg *embed.Config, curls []url.URL, purls []url.URL) {
 	}
 	cfg.InitialCluster = cfg.InitialCluster[1:]
 }
+
+func TestErrOnEmbedClose(t *testing.T) {
```

Comment:
Not sure a full blown test case is necessary for this, a small modification to `TestEmbedEtcd` seems to be enough:
```go
e.Close()
select {
case err := <-e.Err():
    t.Errorf("#%d: unexpected error on close (%v)", i, err)
default:
}
```